Cargo.toml

```diff
@@ -15,20 +15,18 @@ edition = "2021"
 debug = 1
 
 [dependencies]
-actix-cors = "0.5"
-actix-http = "2"
-actix-rt = "1"  # 2+ breaks testing, May need actix-web 4+?
-actix-web = "3"
-actix-web-location = { version = "0.5", features = ["actix-web-v3", "maxmind", "cadence"] }
+actix-cors = "0.6"
+actix-web = "4"
+actix-web-location = { version = "0.7", features = ["actix-web-v4", "maxmind", "cadence"] }
 async-trait = "0.1"
 backtrace = "0.3"
 base64 = "0.13"
 blake3 = "1.0"
-bytes = "1.0"
-cadence = "0.26"
+bytes = "1"
+cadence = "0.29"
 chrono = "0.4"
 docopt = "1.1"
-cloud-storage = { git = "https://github.com/mozilla-services/cloud-storage-rs", branch = "release/0.6.2-create_with_params" } # 0.7+ includes request 0.11, tokio 1.4
+cloud-storage = { git = "https://github.com/mozilla-services/cloud-storage-rs", branch = "release/0.11.1-client-builder-and-params" }
 config = "0.11"
```

Comment:
Didn't want to bump this to 0.13?

Reply:
I was planning to upgrade more things in another PR (like sentry)